frontend/cypress/integration/featureFiles/sidecar_injection.feature

```diff
@@ -11,24 +11,32 @@ Feature: Controlling sidecar injection
 
     Scenario: Override the default policy for automatic sidecar injection by enabling it in a namespace
         Given a namespace without override configuration for automatic sidecar injection
-        When I override the default automatic sidecar injection policy in the namespace to enabled
+        When I visit the overview page
+        And user filters "default" namespace
```

Comment:
I wonder if the fix here is that this line is waiting for kiali to finish loading before attempting to "click" something in the step definition afterward. Basically anytime the tests visit a page we need to make sure that Kiali is done loading the page before trying to interact with it. Otherwise you can end up clicking dom elements that don't have the listeners attached yet or else clicking things and having the page re-render essentially undoing the previous click if you've opened a menu or something. Maybe adding an `ensureKialiFinishedLoading` inside the existing step definitions after the page visit would've fit in more nicely with the way these feature files are written but we haven't been imposing a particular style across the tests and I'm happy that this fixes the flakes.

Reply:
I'm not sure, that may have been a cleaner fix if it works.  Edgar had added instances of `ensureKialiFinishedLoading` in the previous attempt to fix the flakiness, but perhaps we neded them in additional places.  Anyway, since I wasn't looking at that I was tryng to fix my hunch for what the problem may be.  Like you, if the tests pass I'm good.  If we see more failures lets add some more strategic use of `ensureKialiFinishedLoading`.